project/src/main/editor/creature/creature-editor.gd

```diff
@@ -281,7 +252,7 @@ func _alleles_to_mutate() -> Array:
 
 func _on_Breadcrumb_trail_popped(_prev_path: String) -> void:
 	if not Breadcrumb.trail:
-		get_tree().change_scene("res://src/main/ui/menu/LoadingScreen.tscn")
+		get_tree().change_scene("res://assets/main/ui/menu/LoadingScreen.tscn")
```

Comment:
Change this back, and look for other similar mistakes. How was this changed? Did IntelliJ do this?

Reply:
Changed back. Couldn't find any other references.